### PR TITLE
Tusharjagi patch 2

### PR DESCRIPTION
**Pull request by Tusharjagi**

### CODE REVIEW

**Review by devvsakib — DISMISSED** [review]

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/Tusharjagi.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "Tushar Jagi",
+  "displayStatsPublic": true,
```
[the remark this answers is not in the document]
Comment (shubhsharma19) [reply data/Tusharjagi.json:3]: Yes

File: data/Tusharjagi.json [hunk 2]
```diff
@@ -17,6 +17,16 @@
       "name": "Follow me on GitHub",
       "url": "https://github.com/Tusharjagi",
       "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@tusharjagi",
+      "icon": "FaMedium"
```
Comment (priyankarpal) [inline data/Tusharjagi.json:24]: One suggestion please remove the `FaMedium` icon and add the `SiMedium` icon. You can check the changelog of the medium logo from [this blog](https://medium.com/blogging-guide/mediums-new-logo-2020-unfinished-ellipses-logomark-e24c059871e7).
``` suggestion
      "icon": "SiMedium"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
Now it's looking great 🎉

**Review by shubhsharma19 — APPROVED** [review]
Looks good 👍

**Review by mahfuztbt — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉
Great collaboration everyone! 👍

**Pull request merged**